DonutsGo/DonutsGo.WebUI/Components/GreetingInfo.razor

```diff
@@ -0,0 +1,18 @@
+﻿@*
+<div>  
+
+       @*<p>Insert your name</p>
+       <input type ="text"@bind-value="name"/>
+       <button class="btn btn-primary" @onclick="NotifyParent">Add info</button> 
```

Comment:
To be honest I don't really understand the need of this Greeting and GreetingInfo components in your project. I assumed they are used to you get more comfortable with the UI area?